applications/stream-applications-core/stream-applications-security-common/src/test/java/org/springframework/cloud/stream/app/security/common/ReactiveSecurityDisabledManagementSecurityEnabledTests.java

```diff
@@ -38,8 +37,8 @@
 		"spring.main.web-application-type=reactive",
 		"spring.cloud.streamapp.security.enabled=false",
 		"management.endpoints.web.exposure.include=health,info,bindings,env",
+		"management.info.env.enabled=true",
 		"info.name=MY TEST APP"})
```

Comment:
For posterity, should we add a test that shows we can invoke the POST w/o creds when security is disabled?

Reply:
Good catch. I had the same thought.